met-web/src/components/engagement/listing/index.tsx

```diff
@@ -486,16 +488,24 @@ const EngagementListing = () => {
                 </Collapse>
             </Grid>
             <Grid item xs={12}>
-                <MetTable
-                    headCells={headCells}
-                    rows={engagements}
-                    handleChangePagination={(paginationOptions: PaginationOptions<Engagement>) =>
-                        setPaginationOptions(paginationOptions)
-                    }
-                    paginationOptions={paginationOptions}
-                    loading={tableLoading}
-                    pageInfo={pageInfo}
-                />
+                <Suspense fallback={<span />}>
+                    <Await resolve={engagements}>
+                        {(responseData) => {
+                            return (
+                                <MetTable
+                                    headCells={headCells}
+                                    rows={responseData.items}
+                                    handleChangePagination={(paginationOptions: PaginationOptions<Engagement>) =>
+                                        setPaginationOptions(paginationOptions)
+                                    }
+                                    paginationOptions={paginationOptions}
+                                    // loading={tableLoading}
```

Comment:
I think this could be revisited. I believe I disabled it because it was highjacking the page (infinite loading) until I disabled it, but it could be reintegrated.